WordPressKit/DashboardServiceRemote.swift

```diff
@@ -21,17 +32,19 @@ open class DashboardServiceRemote: ServiceRemoteWordPressComREST {
         })
     }
 
-    private func endpoint(for cards: [String], blogID: Int) -> String? {
-        var path = URLComponents(string: "sites/\(blogID)/dashboard/cards-data/")
-
-        let cardsEncoded = cards.joined(separator: ",")
-        path?.queryItems = [URLQueryItem(name: "cards", value: cardsEncoded)]
-
-        guard let endpoint = path?.string else {
-            return nil
+    private func makeQueryParams(cards: [String], deviceId: String) throws -> [String: AnyObject] {
+        let cardsParams: [String: AnyObject] = [
+            "cards": cards.joined(separator: ",") as NSString
+        ]
+        let featureFlagParams = try {
+            let params = FeatureFlagRemote.RemoteFeatureFlagsEndpointParams(deviceId: deviceId)
```

Comment:
I wonder if this struct should still live inside `FeatureFlagRemote` now that it's shared with an unrelated class.